test/runtests.jl

```diff
@@ -1960,6 +2041,179 @@ end
             end
         end
 
+        @testset "Convergence of higher order schemes (autonomous)" begin
+            function f_analytic(u0, p, t)
+                u₁⁰, u₂⁰ = u0
+                a, b = p
+                c = a + b
+                return ((u₁⁰ + u₂⁰) * [b; a] +
+                        exp(-c * t) * (a * u₁⁰ - b * u₂⁰) * [1; -1]) / c
+            end
+
+            # linear model problem - conservative
+            function linmodP!(P, u, p, t)
+                P[1, 2] = u[2]
+                P[2, 1] = 5u[1]
+                return nothing
+            end
+            function linmodP(u, p, t)
+                P = zeros(eltype(u), 2, 2)
+                linmodP!(P, u, p, t)
+                return SMatrix{2, 2}(P)
+            end
+
+            # linear model problem - nonconservative -  in-place
+            function linmodP_noncons!(P, u, p, t)
+                P[1, 1] = u[2] / 2
+                P[1, 2] = u[2] / 2
+                P[2, 1] = 3u[1]
+                P[2, 2] = 2u[1]
+                return nothing
+            end
+            function linmodD_noncons!(D, u, p, t)
+                D[1] = 2u[1]
+                D[2] = u[2] / 2
+                return nothing
+            end
+            function linmodP_noncons(u, p, t)
+                P = zeros(eltype(u), 2, 2)
+                linmodP_noncons!(P, u, p, t)
+                return SMatrix{2, 2}(P)
+            end
+            function linmodD_noncons(u, p, t)
+                D = zeros(eltype(u), 2, 1)
+                linmodD_noncons!(D, u, p, t)
+                return SVector{2}(D)
+            end
+
+            u0 = [Double64(9) / 10; Double64(1) / 10]
+            p = [Double64(5); Double64(1)]
+            tspan = (Double64(0), Double64(1))
+
+            prob_ip = ConservativePDSProblem(linmodP!, u0, tspan, p; analytic = f_analytic)
+            prob_op = ConservativePDSProblem(linmodP, SVector{2}(u0), tspan, SVector{2}(p);
+                                             analytic = f_analytic)
+            prob_ip_noncons = PDSProblem(linmodP_noncons!, linmodD_noncons!, u0, tspan, p;
+                                         analytic = f_analytic)
+            prob_op_noncons = PDSProblem(linmodP_noncons, linmodD_noncons, SVector{2}(u0),
+                                         tspan,
+                                         SVector{2}(p); analytic = f_analytic)
+
+            dts = 0.5 .^ (8:13)
+
+            algs = []
```

Comment:
```suggestion
            algs = MPDeC[]
```
`[]` will result in `Any[]`.